src/nodes/Table/Table.js

```diff
@@ -5,7 +5,7 @@ import TableCell from './TableCell.js'
 import TableHeader from './TableHeader.js'
 import TableHeadRow from './TableHeadRow.js'
 import TableRow from './TableRow.js'
-import { TextSelection } from '@tiptap/pm/state'
+import { Plugin, TextSelection } from '@tiptap/pm/state'
```

Comment:
This change is superfluous now. Removing it also should please the linter. Maybe you can always make sure that your PRs pass the linting tests?